CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+## Bootes - 16-08-2019
+### Added
+- New plugin architecture for the following Asperathos components:
```

Comment:
When adding a new feature, as a user, I would expect a little explanation on what the feature is about, especially in this case when a "new plugin arch" sounds so generic. Just saying something along the lines: "New plugin architecture that allows for the addition of custom plugins based on specific user needs..". Not too much detail, but at least we know what the change is about.

Reply:
Another thing is to remove "Asperathos" from the "Asperathos components" expression. Every change is related to Asperathos, no need to specify each time.